GitUI/CommandsDialogs/FormRebase.Designer.cs

```diff
@@ -28,679 +28,742 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
         {
-            this.components = new System.ComponentModel.Container();
```

Comment:
Was removal of "this." that Designer normally adds an explicit change?
It makes it harder to review.

Reply:
Pain in the ..... designer. I'll revert it

Reply:
So no.  I will not be trying to fix something that was changed by vs.  https://devblogs.microsoft.com/dotnet/winforms-codegen-update/   Yes I tried to folow editorconfig changes to see if it would fix it.  To no avail.  I am not interested in trying to fix something that is autogenerated.  If you want to figure out the right editorconfig that solves the problem and follows the right way, then great.